enhancements/update/accepted-risks.md

```diff
@@ -0,0 +1,476 @@
+    url: https://issues.redhat.com/browse/SDN-3996
+    matchingRules:
+    - type: Always
+    conditions:
+    - status: True  # Apply=True if the risk is applied to the current cluster
+      type: Apply
+      reason: MatchingRule
+      message: The matchingRules[0] matches
+      lastTransitionTime: 2021-09-13T17:03:05Z
+  - name: LeakedMachineConfigBlocksMCO
+    message: MCO update might be blocked if some condition holds
+    url: https://issues.redhat.com/browse/SDN-2212
+    matchingRules:
+    - type: PromQL
+      promql:
+        promql: prom_query
+    conditions:
+    - status: False
+      type: Apply
+      reason: NoMatchingRule
+      message: None of the matching rules matches
+      lastTransitionTime: 2021-09-13T17:03:05Z
+  - name: DualStackNeedsController
+    ...
+  - name: RHELKernelHighLoadIOWait
+    ...
+  conditionalUpdates:
+  - release:
+      version: 4.18.16
+    riskNames:
+    - DualStackNeedsController
+    - OldBootImagesPodmanMissingAuthFlag
+    - RHELKernelHighLoadIOWait
+    risks:  # deprecated by riskNames
+  - release:
+      version: 4.18.17
+    riskNames:
+    - OldBootImagesPodmanMissingAuthFlag
+    risks:  # deprecated by riskNames
+  - release:
+      version: 4.19.1
+    riskNames:
+    - LeakedMachineConfigBlocksMCO
+    risks:  # deprecated by riskNames
+```
+
+When a conditional update is accepted, the names of its associated risks are going to be merged into `clusterversion.status.history.acceptedRisks` which is an existing field before this enhancement. For example, CVO's acceptance of `4.18.17` leads to `OldBootImagesPodmanMissingAuthFlag` being a part of value of `clusterversion.status.history.acceptedRisks`: The wording might look like "The target release ... is exposed to the risks [OldBootImagesPodmanMissingAuthFlag] and accepted by CVO because all of them are considered acceptable".
+
+When a conditional update is blocked, there is a condition in `clusterversion.status.conditions` with `ReleaseAccepted=False`, e.g.,
+
+```yaml
+- lastTransitionTime: 2022-10-11T14:16:13Z
+  message: Payload loaded version="4.18.16" ... is blocked because it is exposed to unaccepted risks [RHELKernelHighLoadIOWait]
+  reason: UnacceptedRisks
+  status: False
+  type: ReleaseAccepted
+```
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+Are there any unique considerations for making this change work with
+Hypershift?
+
+See https://github.com/openshift/enhancements/blob/e044f84e9b2bafa600e6c24e35d226463c2308a5/enhancements/multi-arch/heterogeneous-architecture-clusters.md?plain=1#L282
+
+How does it affect any of the components running in the
+management cluster? How does it affect any components running split
+between the management cluster and guest cluster?
+
+#### Standalone Clusters
+
+Is the change relevant for standalone clusters?
+
+#### Single-node Deployments or MicroShift
+
+How does this proposal affect the resource consumption of a
```

Comment:
Applicable to single-node.  Not applicable to MicroShift, which lacks a ClusterVersion and CVO, and manages updates via RPMs.